### DISCUSSION TITLE
[Suggestion] Promote Tuxemon

### DISCUSSION THREAD

**Discussion by Davidliang11292** [Tuxemon/Tuxemon#2307] [General]
I've noticed that the Tuxemon community seems to be somewhat lacking in vitality, and I believe it's always a good idea to spread the word about Tuxemon. If you feel it's appropriate, please introduce Tuxemon to your family and friends or even within some free software communities (which could include Pokémon ROM hack communities or similar game communities).

I think it's time to breathe new life into Tuxemon. Please feel free to share your suggestions or thoughts on this matter below. Thank you!

**Comment by HippasusTwo** [comment]
Someone could make a post on a fediverse (lemmy, kbin, mastodon) Open Source community advertising it.

It is a bit slow, but I have seen slower communities.

I have been thinking of maybe writing a story or two set in a Tuxemon using universe and publishing it to a free fiction site. Some good places are Royal Road and Space Battles, with Royal Road being very friendly to RPG mechanics like level ups and stats, and Space Battles very friendly to fiction featuring, well, battles, if someone else would like to do this. Some stories can get pretty popular and get lots of attention.
 
My main hesitation about that is that some of the universe details don't seem very settled (did tuxemon always exist or were they invented by humans or is this a many-universe type thing, do tuxemon feel pain, can any of them talk and consent and just happen to find being shoved into little balls very amusing, are we stuck with the evil corporations thing for every region or are some different), so it might end up just being a universe that happens to have tuxemon rather than anything 'canon'. Simplicity that makes for quicker gameplay development (like fewer types which are all equally good) is sometimes at odds with the greater complexity one would desire in story world-building, such as stories about a 'weak type specialist' rising up against the odds to the top, or two societies with very different economic and social policies going to war. For example: One society a wealth-led society where the wealthy simply buy powerful monsters and 'votes' and the poor who want to compete often have to 'pirate' tuxemon, and where tuxemon are considered so disposable you can just throw them in the trash to rot for eternity, vs. another society with a meritocratic focus that more resembles the Pokemon universe with leaders expected to battle their way to the top with carefully raised creatures and tuxemon having the legal right to exit their containers if they really really want to (provided they don't destroy villages when free) but in practice almost never do and also everyone enjoys free health care.

Another storyline possibility is an 'evil' type specialist looking to improve the image of less loved monsters, but none of the basic types really seem suited for getting associated with evil except maybe fire, or metal in a region that has forsaken most technology and gone back to nature, perhaps out of negative reaction toward the corporate-run society. Underdog stories are very popular.

**Reply by Davidliang11292** [reply]
Speaking of which, I have an original script of my own, currently in outline state, but I don't know how to describe the battle scenes, for Tuxemon world Settings, you can visit the Wiki at https://wiki.tuxemon.org/The_World

**Reply by HippasusTwo** [reply]
@Davidliang11292 
You could try looking at the list of techniques to help write the battles.

**Reply by Sanglorian** [reply]
> My main hesitation about that is that some of the universe details don't seem very settled (did tuxemon always exist or were they invented by humans or is this a many-universe type thing, do tuxemon feel pain, can any of them talk and consent and just happen to find being shoved into little balls very amusing, are we stuck with the evil corporations thing for every region or are some different), so it might end up just being a universe that happens to have tuxemon rather than anything 'canon'. 

Would love to see any stories you come up with involving tuxemon. I don't think there's ever going to be a "settled" version of the Tuxemon universe because it's an open source community-created project. I think instead we might see different series, or worlds, or interpretations.

**Reply by HippasusTwo** [reply]
Yeah, I think we'll have to go with the 'multiple universes' thing.
I'll probably try to do a Tuxemon story or two at some point, not necessarily even in the same 'universe'.
